app/src/lib/diff-parser.ts

```diff
@@ -22,6 +22,8 @@ import { getLargestLineNumber } from '../ui/diff/diff-helpers'
 // in which case s defaults to 1
 const diffHeaderRe = /^@@ -(\d+)(?:,(\d+))? \+(\d+)(?:,(\d+))? @@/
 
+export const HiddenBidiCharsRegex = /[\u202A-\u202E]|[\u2066-\u2069]/
```

Comment:
Could we please add a little explanatory comment here since it's being exported?